pkgs/applications/video/stremio/default.nix

```diff
@@ -13,19 +13,19 @@
 
 stdenv.mkDerivation rec {
   pname = "stremio-shell";
-  version = "4.4.165";
+  version = "4.4.168";
 
   src = fetchFromGitHub {
     owner = "Stremio";
     repo = pname;
-    rev = "v${version}";
+    rev = "v${version}-qt6";
```

Comment:
The package is called with qt5, does it have to changed? Perhaps you should use the non qt6 tag
 https://github.com/NixOS/nixpkgs/blob/18c09913154dc18b82f707b73a2e45b128316925/pkgs/top-level/all-packages.nix#L12710

https://github.com/Stremio/stremio-shell/commit/b4385fb63b99e8e8ab47d0716355378fdc5de8f0#commitcomment-144853058
https://github.com/Stremio/stremio-shell/issues/380
Another nixpkgs user asked about qt6 but got no replies

Reply:
Good catch! For now I've converted the PR to update to the latest qt5 release.
The qt6 will need more significant modifications, so it might be a good idea to track that in a separate commit.